src/resource_clients/dataset.ts

```diff
@@ -191,6 +240,7 @@ export interface Dataset {
     stats: DatasetStats;
     fields: string[];
     generalAccess?: STORAGE_GENERAL_ACCESS | null;
+    urlSigningSecretKey?: string | null;
```

Comment:
We're also now returning `publicItemsUrl` in the dataset response, right? Maybe let's add it to the type?